packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart

```diff
@@ -1124,11 +1169,20 @@ PlatformMapConfiguration _platformMapConfigurationFromMapConfiguration(
     indoorViewEnabled: config.indoorViewEnabled,
     trafficEnabled: config.trafficEnabled,
     buildingsEnabled: config.buildingsEnabled,
-    cloudMapId: config.cloudMapId,
+    markerType: _platformMarkerTypeFromMarkerType(config.markerType),
+    mapId: config.mapId,
     style: config.style,
   );
 }
 
+PlatformMarkerType? _platformMarkerTypeFromMarkerType(MarkerType? markerType) {
```

Comment:
Github won't let me add a comment on the line, but `collisionBehavior` needs to be added to `_platformMarkerFromMarker` function above

Reply:
Great catch, thanks! I've added a test for `updateMarkers` that works with `AdvancedMarker`s and a check for `collisionBehavior` there. Do you think it's worth adding a test just for `_platformMarkerTypeFromMarkerType`?